### PR TITLE
link to matrix.org/security-disclosure-policy in help screen

### PR DESCRIPTION
**Pull request by benparsons**
Screenshot 2020-03-05 at 14 19 21

### CODE REVIEW

**Comment by turt2live** [comment]
@benparsons there's a few things that need to be done before we can merge this:
* [ ] Run `yarn i18n` to fix the CI
* [ ] Put a screenshot in the PR description so that the design team can review
* [ ] Request review from `@matrix-org/design` (to review copy, placement, etc - we're up against a release schedule which means we can't realistically merge and fix later).

For future PRs, please also request review of the riot-web team as we are likely to miss the PR otherwise.

**Comment by aaronraimist** [comment]
Hmm. Should Riot/New Vector have it's own policy?

If not, it feels weird to call it the Matrix.org Security Disclosure Policy, as if it is relating to the homeserver. Maybe the "Matrix Security Disclosure Policy" or the "Matrix.org Foundation Security Disclosure Policy"

File: src/components/views/settings/tabs/user/HelpUserSettingsTab.js [hunk 1]
```diff
@@ -225,6 +225,15 @@ export default class HelpUserSettingsTab extends React.Component {
                                 {_t("Clear cache and reload")}
                             </AccessibleButton>
                         </div>
+                        {
+                            _t( "To submit a security-related bug report, please read the Matrix.org " +
+                                "<a>Security Disclosure Policy</a>.", {},
+                                {
+                                    'a': (sub) =>
+                                        <a href="https://matrix.org/security-disclosure-policy/"
+                                        rel='noreferrer noopener' target='_blank'>{sub}</a>,
```
Comment (t3chguy) [inline src/components/views/settings/tabs/user/HelpUserSettingsTab.js]: can this use double quotes like the line above it for consistency and my sanity :D

Comment (benparsons) [reply src/components/views/settings/tabs/user/HelpUserSettingsTab.js]: yes

**Comment by nadonomy** [comment]
> Hmm. Should Riot/New Vector have its own policy?
> 
> If not, it feels weird to call it the Matrix.org Security Disclosure Policy, as if it is relating to the homeserver. Maybe the "Matrix Security Disclosure Policy" or the "Matrix.org Foundation Security Disclosure Policy"

I think principally you're correct, but considering the security disclosure policy is hosted at matrix.org, I'm not sure the current implementation of "Matrix.org Security Disclosure Policy" is too misleading— especially considering this is just a shorthand description for users to click through and gain more context. "Matrix Security Disclosure Policy" would be indicative of all of the uses of Matrix everywhere, which isn't the case. And I think it's OK for Matrix.org to equal Matrix.org Foundation.

@benparsons the content proposed here looks good to me, but it's highlighting that funky padding has made it through to production where it's hard to discern an information hierarchy. I'll DM you to figure out some simple iteration.

**Comment by benparsons** [comment]
(updated image preview)

**Comment by jryans** [comment]
@benparsons Looks like you need to re-run `yarn i18n`.

**Pull request merged**